CDP4CrossViewEditor/Views/CrossViewDialog.xaml

```diff
@@ -21,12 +21,19 @@
             <Grid.RowDefinitions>
                 <RowDefinition Height="Auto" />
                 <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
             </Grid.RowDefinitions>
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="*" />
             </Grid.ColumnDefinitions>
-            <uc:ElementThingSelector Grid.Row="0" Mode="ElementBase" VerticalAlignment="Stretch" x:Name="ElementThingSelector" />
-            <uc:ParameterThingSelector Grid.Row="1" Mode="ParameterBase" VerticalAlignment="Stretch" x:Name="ParameterThingSelector" />
+            <dxlc:LayoutControl View="Group"
```

Comment:
One thing I suggest we do is move the checkbox and the text to the bottom of the dialog, and align it to the right, so it jumps out more when clicking OK / Cancel. Where it is now (upper left) makes it very easy to ignore/disregard.

Reply:
I'm not saying this is the right way, just a suggestion that might be worth implementing _if_ you agree.